__tests__/app.test.js

```diff
@@ -80,95 +81,98 @@ describe("GET /api/articles/:article_id", () => {
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('created_at', { descending: true })
-    })
-  })
-  
-  test("200: Accepts a sort_by query, which defaults to the created at data", () => {
-    return request(app)
-    .get("/api/articles?sort_by")
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('created_at', { descending: true })
-    })
-
-  })
+      })
 
-  test("200: Accepts a sort_by query for created at data", () => {
-    return request(app)
-    .get("/api/articles?sort_by=created_at")
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('created_at', { descending: true })
-    })
-  })
+      test("200: Accepts a sort_by query for created at data", () => {
+        return request(app)
+        .get("/api/articles?sort_by=created_at")
+        .expect(200)
+        .then(({body : {articles }}) => {
+          expect(articles).toBeSortedBy('created_at', { descending: true })
+        })
+      })
 
-  test("200: Accepts a sort_by query for topics", () => {
-    return request(app)
-    .get("/api/articles?sort_by=topic")
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('topic', { descending: true })
-    })
-  })
+      test("200: Accepts a sort_by query for topics", () => {
+        return request(app)
+        .get("/api/articles?sort_by=topic")
+        .expect(200)
+        .then(({body : {articles }}) => {
+          expect(articles).toBeSortedBy('topic', { descending: true })
+        })
+      })
 
-  test("200: Accepts a sort_by query for authors", () => {
-    return request(app)
-    .get("/api/articles?sort_by=author")
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('author', { descending: true })
-    })
-  })
+      test("200: Accepts a sort_by query for authors", () => {
+        return request(app)
+        .get("/api/articles?sort_by=author")
+        .expect(200)
+        .then(({body : {articles }}) => {
+          expect(articles).toBeSortedBy('author', { descending: true })
+        })
+      })
 
-  test("200: Accepts a sort_by query for votes", () => {
-    return request(app)
-    .get("/api/articles?sort_by=votes")
-    .expect(200)
-    .then(({body : {articles }}) => {
-      expect(articles).toBeSortedBy('votes', { descending: true })
-    })
+      test("200: Accepts a sort_by query for votes", () => {
+        return request(app)
+        .get("/api/articles?sort_by=votes")
+        .expect(200)
+        .then(({body : {articles }}) => {
+          expect(articles).toBeSortedBy('votes', { descending: true })
+        })
+      })
   })
-
-  test("200: Accepts an order query which defaults to descending", () => {
+  describe("?order", () => {
+    test("200: Accepts an order query which defaults to descending", () => {
     return request(app)
     .get("/api/articles?order")
```

Comment:
Again, I would just show that when we go to `/api/articles` it will order the articles in descending order.